pytest.ini~

```diff
@@ -0,0 +1,3 @@
+[pytest]
+base_url = http://localhost:8000
+sensitive_url = osmaxx.hsr.ch
```

Comment:
you can also add `*~` to your .gitignore file to exclude all files ending with `~`